allure-report-face/src/index.js

```diff
@@ -0,0 +1,19 @@
+import App from './app';
+import $ from 'jquery';
+import Backbone from 'backbone';
+import allurePlugins from './pluginApi';
+import './plugins/default';
+import './plugins/defects';
+import './plugins/environment';
+import './plugins/xunit';
+import './plugins/behaviors';
+import './plugins/graph';
+import './plugins/timeline';
+
+import DescriptionView from './plugins/testcase-description/DescriptionView';
+allurePlugins.addTestcaseBlock(DescriptionView, {position: 'before'});
+
+App.start();
+Backbone.history.start();
+
+window.jQuery = $;
```

Comment:
Also, there is a Chrome Extension - [Marionette-inspector](https://github.com/marionettejs/marionette.inspector) which also needs globally available that libraries